Language/Haskell/GhcMod/Gap.hs

```diff
@@ -293,11 +293,11 @@ setDeferTypeErrors dflag = dopt_set dflag Opt_DeferTypeErrors
 setDeferTypeErrors = id
 #endif
 
-setWarnTypedHoles :: DynFlags -> DynFlags
+setDeferTypedHoles :: DynFlags -> DynFlags
 #if __GLASGOW_HASKELL__ >= 708
-setWarnTypedHoles dflag = wopt_set dflag Opt_WarnTypedHoles
+setDeferTypedHoles dflag = gopt_set dflag Opt_DeferTypedHoles
 #else
-setWarnTypedHoles = id
+setDeferTypedHoles = id
```

Comment:
Okay, I can put that back. It's very difficult to navigate how the flags are intended to work within GHC because of the private API issues.